lib/LedgerSMB/Scripts/import_csv.pm

```diff
@@ -12,21 +12,24 @@ data.
 =cut
 
 package LedgerSMB::Scripts::import_csv;
-use Moose;
+use strict;
+use warnings;
+
+use List::MoreUtils qw{ any };
+
 use LedgerSMB::Template;
 use LedgerSMB::Form;
-use strict;
+use LedgerSMB::Setting;
 
-my $default_currency = 'USD';
 our $cols = {
    gl       =>  ['accno', 'debit', 'credit', 'source', 'memo'],
    ap_multi =>  ['vendor', 'amount', 'account', 'ap', 'description',
                  'invnumber', 'transdate'],
    ar_multi =>  ['customer', 'amount', 'account', 'ar', 'description',
                  'invnumber', 'transdate'],
-   timecard =>  ['employee', 'projectnumber', 'transdate', 'partnumber',
-                 'description', 'qty', 'noncharge', 'sellprice', 'allocated',
-                'notes'],
+   timecard =>  ['employee', 'business_unit_id', 'transdate', 'partnumber',
```

Comment:
Instead of using the business_unit_id, I'd rather use the class and control code here and look that up during processing to get to the business_unit_id: much better for the user.